.github/workflows/docker-test.yml

```
@@ -112,4 +143,15 @@ jobs:
           echo "### 🔧 Local Testing" >> $GITHUB_STEP_SUMMARY
           echo "\`\`\`bash" >> $GITHUB_STEP_SUMMARY
           echo "docker run -p 8080:8080 ${{ steps.meta.outputs.tags }}" >> $GITHUB_STEP_SUMMARY
-          echo "\`\`\`" >> $GITHUB_STEP_SUMMARY 
\ No newline at end of file
+          echo "\`\`\`" >> $GITHUB_STEP_SUMMARY
+
+    secrets:
```

actionlint (1.7.8)

"secrets" is only available for a reusable workflow call with "uses" but "uses" is not found in job "build-and-test"

(syntax-check)